### ISSUE TITLE
DatePicker 3/29 testing followups

### ISSUE THREAD

**Issue by LFDanLu** [adobe/react-spectrum#2991]
# 🐛 Bug Report 

- The stylized sandbox examples in DatePicker docs break when you try to move focus between the datepicker spin buttons via arrow keys
- NVDA doesn't announce the spin buttons within the readonly DatePicker as readonly. Note that the timezone spinbutton is announced as readonly so maybe there is something done there that can be done for the readonly datepicker spin buttons

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |
| Browser          |
| Operating System |

## 🧢 Your Company/Team

RSP

**Comment by devongovett** [comment]
First one is fixed now that I've updated the code sandboxes.

NVDA issue is actually a chrome issue I think. When an element is contenteditable, it doesn't seem to announce anything about the spin button role at all. NVDA and Firefox does though.

Gonna close.
